orders-core/src/test/java/io/helidon/examples/sockshop/orders/TestCartsClient.java

```diff
@@ -0,0 +1,18 @@
+package io.helidon.examples.sockshop.orders;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import static io.helidon.examples.sockshop.orders.TestDataFactory.items;
+
+@Mock
+@ApplicationScoped
+public class TestCartsClient implements CartsClient {
```

Comment:
Can we use Mockito mocks in combination with CDI producers, instead of creating a separate class for each mock?

Reply:
Let's address separately.